.github/settings.yml

```diff
@@ -0,0 +1,14 @@
+# The settings are inherited from common repository repo-settings (same file name and location)
+# You can append settings that are deeply merged with the inherited settings.
+#
+# When you have installed the GitHub App "repo-settings" in this repository,
+# any change of this settings.yml file is detected by the GitHub App and
+# the settings of this repository are updated immediately.
+#
+_extends: repo-settings:.github/common-settings-v2.yml
+
+# repo-specific settings
+#
+repository:
+  name: "terraform-ibm-hpcs"
+  description: "Create and configure an IBM Cloud Hyper Protect Crypto Services instance."
```

Comment:
would you mind adding this:
`topics: core-team, terraform, ibm-cloud, terraform-module, hpcs, hyper-protect, kms`